src/wallet/rpc/backup.cpp

```diff
@@ -93,6 +93,20 @@ static void RescanWallet(CWallet& wallet, const WalletRescanReserver& reserver,
     }
 }
 
+static void EnsureBlockDataFromTime(interfaces::Chain& chain, const CWallet& wallet, int64_t timestamp)
```

Comment:
Why create a function for this? Would this be easier to reason about if it was inline?

Reply:
I've seen many instances of `Ensure(.*)` functions and it also helps to reduce the length of the parent function I suppose.